setup.py

```diff
@@ -22,7 +22,7 @@
     extension = Extension(
         'uharfbuzz._harfbuzz',
         libraries = ['harfbuzz'],
-        library_dirs = ['/usr/include/harfbuzz'],
+        include_dirs = ['/usr/include/harfbuzz'],
```

Comment:
This does not make sense, we shouldn’t be using headers from system. This should point to `harfbuzz/src` instead.